src/biokbase/narrative/common/kvp.py

```diff
@@ -34,6 +32,7 @@ def parse_kvp(msg, record, text_sep=" "):
     for n, v, vq, txt in KVP_EXPR.findall(msg):
         if n:
             if vq:
+                # N.b. probable coding error: v is used in the "for" loop
```

Comment:
No tests => did not fix.

Reply:
I'm reasonably certain that the old log_proxy / log_common stack doesn't work, and hasn't worked in a while. That's likely not called anywhere important.

Eventually, I'd like to remove it and update the logging code, so this isn't worth the time to test and fix.